app/views/layouts/step.html.erb

```diff
@@ -12,7 +12,6 @@
     </h4>
   </div>
   <%= render 'shared/debug' %>
-  <%= render 'shared/flashes' %>
```

Comment:
removed this because I realized that due to the way our layout is, flashes dont render properly on step pages anyway

Reply:
👍